mounts_docker.go

```diff
@@ -0,0 +1,116 @@
+package testcontainers
```

Comment:
NIT: I'd rename this file to `docker_mounts.go` as it's used as `..DockerMounts...` everywhere.